cli_tools/common/vdisk/file_inspector.go

```diff
@@ -0,0 +1,124 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vdisk
```

Comment:
Hmm, sounds like it was confusing? Any suggestions? @dntczdx what do you think?

Reply:
to me vdisk is too short. Even VirtualDisk is not very good since we almost always call it "disk image" in our projects. I think we never use the term "virtual". It sound slike we are bringing some new concepts.

Reply:
Thanks Tony and Zoran.  Here's a quick proposal
1. Change the package to `imagefile`
2. Change the struct to `Metadata`

With these changes, package users would see `imagefile.Metadata`.

A couple of other proposals
1. `diskfile.Metadata` 
2. `virtualdisk.FileMetadata`

These are just proposals! What do you guys think? Any ideas?

Reply:
`imagefile` or `diskfile` sound good to me. 

`virtual` might not be necessary from the context of file as it's obvious it's not physical.

Reply:
Cool. Updated the package name to `imagefile`